src/Controllers/LoginSessionController.php

```diff
@@ -88,6 +94,7 @@ private function jsonResponse(array $response, int $code = 200): HTTPResponse
     {
         return HTTPResponse::create(json_encode($response))
             ->addHeader('Content-Type', 'application/json')
+            ->addHeader('Cache-Control', 'no-cache')
```

Comment:
Added this extra bit to be 100% we never cache the response.

Reply:
Would `HTTPCacheControlMiddleware::singleton()->disableCache()` work in this context?  If so that's preferable since it's the standard mechanism for this

Other wise the value here should be `no-store` instead which means "The response may not be stored in any cache" - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Reply:
Very good suggestion ... done. I also added a unit test to validate tho "Cache-Control" header.